### PR TITLE
Feat: Extra Features

### PR DESCRIPTION
**Pull request by aniketxparihar**
Features Added

Comment
Search
Upload Media Posts

### CODE REVIEW

File: src/Components/CommentCard/CommentCard.jsx [hunk 1]
```diff
@@ -67,6 +73,25 @@ const CommentCard = ({ comment }) => {
...
         </div>
+        <span
+          className="material-symbols-rounded  ml-auto mr-8"
+          onClick={() =>
+            editComment(comment?.postId, comment?._id, authToken)
+          }
+          style={{ color: themeObject.text }}
+        >
+          edit
+        </span> */}
```
Comment (ScoobyDubeyDo) [inline src/Components/CommentCard/CommentCard.jsx]: **suggestion**: delete unwanted code if not needed

**Review by ScoobyDubeyDo — APPROVED** [review]
- Dividing the code into multiple components
- Good PR
- Liked the barrel export
- easily understandable code

File: src/Components/CommentCard/CommentCard.jsx (hunk 1 above)
Comment (aniketxparihar) [reply src/Components/CommentCard/CommentCard.jsx]: Thanks for reminding this.

**Review by shobit1337 — APPROVED** [review]

**Comment by shobit1337** [comment]
Modular code and UI looks cool, Good work @aniketxparihar

**Pull request merged**